data/json/npcs/exodii/exodii_merchant_Rubik_stub.json

```diff
@@ -0,0 +1,241 @@
+[
+  {
+    "type": "npc",
+    "id": "exodii_merchant",
+    "name_suffix": "Cyborg",
+    "class": "NC_EXODII_TYPE_1_Merchant",
+    "name_unique": "Rubik",
+    "attitude": 0,
+    "mission": 3,
+    "chat": "TALK_EXODII_MERCHANT",
+    "faction": "exodii"
+  },
+  {
+    "type": "npc_class",
+    "id": "NC_EXODII_TYPE_1_Merchant",
+    "name": { "str": "Cyborg, type 1" },
+    "job_description": "I'm a cyborg.",
+    "traits": [
+      { "trait": "EXODII_BODY_1" },
+      { "//": "This needs to give the exodii a unique sprite and hide any worn or wielded overlays" }
+    ],
+    "common": false,
+    "bonus_str": { "rng": [ 0, 6 ] },
+    "bonus_dex": { "rng": [ 0, 6 ] },
+    "bonus_int": { "rng": [ 0, 2 ] },
+    "bonus_per": { "rng": [ 0, 4 ] },
+    "shopkeeper_item_group": "EXODII_SHOP_Tier0",
+    "carry_override": "NC_EXODII_TYPE_1_carried",
+    "worn_override": "EMPTY_GROUP",
+    "skills": [
+      {
+        "skill": "ALL",
+        "level": { "mul": [ { "one_in": 3 }, { "sum": [ { "dice": [ 2, 2 ] }, { "constant": 0 }, { "one_in": 4 } ] } ] }
+      },
+      { "skill": "mechanics", "bonus": { "one_in": 2 } },
+      { "skill": "electronics", "bonus": { "rng": [ 0, 2 ] } },
+      { "skill": "speech", "bonus": { "rng": [ 5, 8 ] } }
+    ]
+  },
+  {
+    "type": "mutation",
+    "id": "EXODII_BODY_1",
+    "flags": [ "Exodii_Body" ],
+    "name": { "str": "Exodii cyborg framework - style 1a" },
+    "points": 0,
+    "visibility": 4,
+    "ugliness": 4,
+    "mixed_effect": true,
+    "description": "Your brain and some of your organs are shielded in a nice tough bipedal walker frame.",
```

Comment:
"tough" implies a bit of armor, at least to me. If this is meaning the HP adjustment, perhaps "resilient"?

Reply:
It should have armour as well, I just haven't gotten around to it.